sauce/features/budget/enter-to-move/index.js

```diff
@@ -43,7 +39,19 @@ export class EnterToMove extends Feature {
       changedNodes.has(CATEGORY_DROPDOWN) &&
       changedNodes.has(BUTTON_PRIMARY)
```

Comment:
This check against `changedNodes` is different than the check for `MOVE_POPUP` because they occur at different changes. The `MOVE_POPUP` is only for checking that the modal is open, but this check is to see if the OK button has been enabled. Hence why these checks aren't combined as was suggested.

Reply:
Yeah that makes sense.